Bender.local

```diff
@@ -3,10 +3,10 @@
 # SPDX-License-Identifier: SHL-0.51
 
 overrides:
-  axi:                  { git: https://github.com/pulp-platform/axi.git                   , version: 0.39.0 }
+  axi:                  { git: https://github.com/pulp-platform/axi.git                   , version: 0.39.1-beta }
```

Comment:
I am already doing it in #111; but if you need to point to latest `pulp_cluster` (I know I pushed on top last week for the dedicated PR on CDC sync parameter propagation), we can also do it here :)

Reply:
If you don't mind, let's do the alignment here so we can bump more than one IP at once in the same PR!

Reply:
agreed